oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/endpoint/DefaultClientCredentialsTokenResponseClient.java

```diff
@@ -129,4 +132,26 @@ public void setRestOperations(RestOperations restOperations) {
 		this.restOperations = restOperations;
 	}
 
+	private static final class ClientAuthenticationMethodVerifyingRequestEntityConverter
+			implements Converter<OAuth2ClientCredentialsGrantRequest, RequestEntity<?>> {
```

Comment:
Looks like the reusable class was only used in `DefaultAuthorizationCodeTokenResponseClient`?

Reply:
Good catch, thanks.